services/accumulo-utils/pom.xml

```diff
@@ -30,6 +30,11 @@
                 <artifactId>authorization-api</artifactId>
                 <version>${version.authorization-api}</version>
             </dependency>
+            <dependency>
```

Comment:
This is a huge no-no. We don't want any dependencies in the microservices on the datawave query code as-is. If there is a common dependency, then it needs to be moved to one of the appropriate utils packages instead.

Reply:
This was only done because of Span and Trace. They were dropped so I just stuck them in datawave.webservice.query.runner since that package used them the most... We shouldn't be using them at all but I don't know where they should go if we have to copy them over.